python/datasources/cdc_miovd.py

```diff
@@ -0,0 +1,139 @@
+"""
+This documentation outlines the procedure for querying the homicide and suicide reports
+from the CDC MIOVD datasource.
+
+Manual Query Steps:
+1. Open your web browser and navigate to the CDC data portal:
+    - (https://data.cdc.gov/Injury-Violence/Mapping-Injury-Overdose-and-Violence-County/psx4-wq38/data_preview).
+2. Select "Actions" and choose "Query Data."
+4. Filter the dataset by intent to display either "FA_Homicide" for homicide reports
+   or "FA_Suicide" for suicide reports.
+
+Note: If the provided link does not work, please refer to the URL in the comment section below.
+
+URL for manual access:
+https://data.cdc.gov/Injury-Violence/Mapping-Injury-Overdose-and-Violence-County/psx4-wq38/about_data
+
+Last Updated: 6/3/2025
+"""
+
+import pandas as pd
+
+from datasources.data_source import DataSource
+from ingestion import dataset_utils, gcs_to_bq_util, merge_utils
+from ingestion import standardized_columns as std_col
+from ingestion.constants import CURRENT, HISTORICAL
+
+
+class CDCMIOVDData(DataSource):
+    # MIOVD constants

```

Comment:
this comment is confusing to me; usually we do this term/pattern to declare the source data constants, like `FA_Homicide` etc. Maybe we can remove this comment or change it to like, "File loading constants" or something?